src/System Application/App/ClientExtensions/src/CameraBarcodeScannerProviderAddIn.al

```diff
@@ -0,0 +1,60 @@
+namespace System.ClientExtensions;
+
+// The control add-in for the camera barcode scanner provider.
+controladdin CameraBarcodeScannerProviderAddIn
```

Comment:
Isn't CameraBarcodeScannerProvider enough?

Reply:
not really, it will conflict with the current DotNet class name